1_basics.ipynb

```diff
@@ -183,18 +350,53 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Check that RViz has started on the simulator screen. In RViz, you can visualize various sensor information mounted on the robot.\n",
```

Comment:
"Check that RViz has started **in** the simulator screen. In RViz, you can visualize various sensor information **from the sensors that are** mounted on the robot.\n",